resource-service/src/component-tests/java/uk/gov/gchq/palisade/component/resource/model/AuditErrorMessageTest.java

```diff
@@ -15,30 +15,26 @@
  */
 package uk.gov.gchq.palisade.component.resource.model;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.json.JsonTest;
-import org.springframework.boot.test.json.JacksonTester;
-import org.springframework.boot.test.json.JsonContent;
-import org.springframework.boot.test.json.ObjectContent;
 import org.springframework.test.context.ContextConfiguration;
 
 import uk.gov.gchq.palisade.Context;
 import uk.gov.gchq.palisade.service.resource.model.AuditErrorMessage;
 
 import java.io.IOException;
-import java.util.HashMap;
 import java.util.Map;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertAll;
 
 @JsonTest
 @ContextConfiguration(classes = {AuditErrorMessageTest.class})
 class AuditErrorMessageTest {
```

Comment:
If this is just using an ObjectMapper, we don't need this to be a `@JsonTest` (which gives you the now-unused JacksonTester bean)